src/Cake/Infrastructure/Scripting/RoslynScriptSession.cs

```diff
@@ -48,6 +60,11 @@ public sealed class RoslynScriptSession : IScriptSession
             ReferencePaths = new HashSet<FilePath>(PathComparer.Default);
             References = new HashSet<Assembly>();
             Namespaces = new HashSet<string>(StringComparer.Ordinal);
+
+            var cacheEnabled = configuration.GetValue(Constants.Cache.Enabled) ?? "false";
+            _scriptCacheEnabled = cacheEnabled.Equals("true", StringComparison.OrdinalIgnoreCase);
+            _regenerateCache = host.Context.Arguments.HasArgument("regeneratecache");
```

Comment:
`invalidate-cache`

Reply:
Would also be good to avoid "magic strings". Do we have a `Constants.cs` where these should reside?